### DISCUSSION TITLE
[Mobile] Library page shows map and on device albums when the features are disabled.

### DISCUSSION THREAD

**Discussion by natedawg** [immich-app/immich#13537] [Feature Request]
### The bug

When the Map is disabled from the server admin page it's expected that the Places album would be disabled on mobile too. 
Same with the mobile "On this device" album. If the file access permission is blocked the blank album should not be shown.

### The OS that Immich Server is running on

Docker on debian

### Version of Immich Server

v1.118.0

### Version of Immich Mobile App

v1.118.0

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
NA
```

### Your .env content

```Shell
NA
```

### Reproduction steps

1. Disable the map from the immich administration settings and confirm it is disabled on the web.
2. Open the library page on mobile and the Places album is still enabled. 
3. Do not allow file acces permissions on mobile.
4. Open the library page on mobile and the "On this device" album is still visable and blank.   
...

**Comment by danieldietzler** [comment]
For one thing, those features cannot be enabled/disabled in the server admin settings but in the web user settings. That those settings currently don't apply to mobile is expected, and adding that is a feature request

**Reply by natedawg** [reply]
No the map toggle is on the administration page (Administration > Settingsv> Map & GPS & Settings). It's expected that those settings would apply to all users and devices. The on device album is also a bug. Why show a blank album when the device is unable populated it with no permissions?

**Reply by danieldietzler** [reply]
Ah those settings should ideally apply to all devices, that is correct. For the latter I think that's a personal opinion (I personally find the empty album more intuitive). However, it would be helpful if you could split those two feature requests up so that we can link them accordingly in respective PRs.
